GATK_CNV_Mutect2/GATK4_CNV/GATK4_CNV.wdl

```diff
@@ -655,12 +655,20 @@ task DenoiseReadCounts {
         set -e
         export GATK_LOCAL_JAR=${default="/root/gatk.jar" gatk4_jar_override}
 
+        if [[ -f "${read_count_pon}" ]]; then
         gatk --java-options "-Xmx${command_mem_mb}m" DenoiseReadCounts \
             --input ${read_counts} \
             --count-panel-of-normals ${read_count_pon} \
             ${"--number-of-eigensamples " + number_of_eigensamples} \
             --standardized-copy-ratios ${entity_id}.standardizedCR.tsv \
             --denoised-copy-ratios ${entity_id}.denoisedCR.tsv
+        fi
+
+        gatk --java-options "-Xmx${command_mem_mb}m" DenoiseReadCounts \
```

Comment:
Yes. The changes I made here is to make the `read_count_pon` input optional to GATK4_CNV.wdl. So, the workflow will run regardless the presence of `read_count_pon`. The motivation of this change is because ticket [TAG1580](https://github.com/broadinstitute/TAG/issues/1580). We didn't have a `read_count_pon` for PANCAN but we have an interval list for PANCAN. The original WDL is not able to run the data in provided in TAG1580 because it required a `read_count_pon` input.

Reply:
Also this wdl is performing as expected. I used it in the TAG1580 analysis without running into any error.